packages/kbn-sharedux-services/BUILD.bazel

```diff
@@ -0,0 +1,126 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
```

Comment:
Is this something auto-generated?

Reply:
@spalger wrote a script, but we have to edit it in some cases: https://github.com/elastic/kibana/pull/127095